### DISCUSSION TITLE
Bots

### DISCUSSION THREAD

**Discussion by Huckelbarry** [community/community#74766] [Programming Help]
### Select Topic Area

Question

### Body

I'm 54 and not too computer savvy. I have been trying to build a bot for weeks. The site said 5 minutes. Lol. I'm officially asking for help. I have one started but I don't know what I'm doing. Any help will be appreciated.

**Comment by prawi** [comment]
If you could share more details about the platform or framework you've started with or your bot's intended purpose, I can provide more tailored guidance.
